app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -182,6 +159,22 @@ public void onClick(View v) {
     }
 
     /**
+     * Set response when clicking in the bottom navigation
```

Comment:
This isn't too descriptive. It should tell us what happens on clicking the BottomBar (i.e Shows/hide the visualization fragment (replace this with : {@link [name of the visualization fragment] on click)